### DISCUSSION TITLE
Stardock Curtains + Fluent (Possible fix?)

### DISCUSSION THREAD

**Discussion by Revonlieke** [purogamer/Fluent-for-Steam#69] [Ideas]
So I am using Stardock Curtains and I'm really enjoying the look of Fluent.

I use Stardock Curtains with a Windows theme to have rounded corners on most of my default explorer and browser windows.
Currently on Win10 (Since Win11 is not an option for me)

Stardock curtains does round out the bottom area of the window on Fluent (as it does on default steam skins too), it replaces the "shadow" at the bottom with a rounded bar.

The top section however is sharp, from my understanding I would need to have a way to replace the "titlebar" with  system default title bar. I use this method on browsers for example.

 Now Fluent doesn't have the actual "titlebar" present. Would it be possible to bring it back, but in a way where it would use the systems titlebar instead? This could possibly make Fluent have rounded corners even on Win10 when using Stardock Curtains & a custom Windows10 theme with rounded corners. Like: https://www.deviantart.com/niivu/art/BIB3-for-Windows-886441919
 
 Here's a look at how it works for browser title bars - https://www.deviantart.com/nystrata/art/Windows-Custom-Visual-Styled-Titlebars-in-Browsers-916187416
 
 Hope it's possible in some miraculous way.
 Thank you!

**Comment by redsigma** [comment]
Does it work with default steam ?

**Reply by Revonlieke** [reply]
No, exact same issue. The default steam has the top bar present, which has "Steam", "View", "Friends", "Games" & "Help" on it, but still has sharp corners, most likely due to being steams own titlebar, or it's part of the entire header portion. (I do not know)

Basically it would need to be replaced by the system titlebar which you see by opening notepad or just any file explorer for curtains to do it's trick.

**Reply by Revonlieke** [reply]
Been thinking about it and judging by how it works on the browsers, it just seems to set the custom titlebar of the software invisible/transparent.

The stardock curtain must just be underneath the custom one.
